composer/trainer/devices/device_tpu.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""The TPU device used for training."""
+
+from __future__ import annotations
+
+import logging
+from contextlib import contextmanager
+from typing import Generator, TypeVar, Union
+
+import torch
+import torch_xla
```

Comment:
This will need to be a conditional import.

Reply:
fixed